### DISCUSSION TITLE
Issues

### DISCUSSION THREAD

**Discussion by Liquefy7822** [community/community#49774] [GitHub Education]
I am not able to sign up with my school email. My organisation is there but not supported for my batch. There is one for the year 2021 but not my 2022 batch. What can I do?

**Comment by samus-aran** [comment]
Hi @Liquefy7822 

We're happy you're here. 👍 
You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by Liquefy7822** [reply]
Hi I have etried reaching put to my school too and am awaiting a response. Apparently it is a problem on my schools side
